test/e2e/testdata/artifactgc/artgc-artifact-not-written.yaml

```
@@ -0,0 +1,28 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: testing-artifact-not-present-
```

```suggestion
  generateName: artgc-artifact-not-written-
```

these normally match the filename by convention, no?